books/tests/test_models.py

```diff
@@ -5,7 +5,6 @@
 
 @pytest.mark.django_db
 class TestBook:
-    def test_model(self, book_owner1):
-        user1 = book_owner1
-        obj = mixer.blend('books.Book', title="A book")
-        assert obj.__str__() == "A book", 'Should return title'
+    def test_model(self):
+        book = mixer.blend('books.Book', title="A book")
+        assert book.__str__() == "A book", 'Should return title'
```

Comment:
Not a real issue, just usual practice:
`__str__` and other magic functions(like `__init__`) are not really meant to be called directly. To test `__str__` you would normally do:
```python
assert str(book) == "..."
```